napari/utils/settings/_tests/test_settings.py

```diff
@@ -162,7 +162,6 @@ def test_settings_env_variables(tmp_path, monkeypatch):
     value = 'light'
     monkeypatch.setenv('NAPARI_THEME', value)
     settings = SettingsManager(tmp_path, save_to_disk=True)
-    assert CORE_SETTINGS[0]().theme == value
```

Comment:
removed this here cause it feels repetitive... but if you wanted to keep it in, you'd need to do somethiing like

```python
appearance: AppearanceSettings = CORE_SETTINGS[0]()
assert appearance.theme == value
```